src/lib/stack-client.js

```diff
@@ -0,0 +1,236 @@
+/* global __TARGET__ */
+/* eslint-env browser */
+
+import CozyClient from 'cozy-client'
+import initializeRealtime from './realtime'
+
+import {
+  ForbiddenException,
+  ServerErrorException,
+  NotFoundException,
+  MethodNotAllowedException,
+  UnavailableStackException,
+  UnavailableSettingsException,
+  UnauthorizedStackException
+} from './exceptions'
+
+import getIcon from './icon'
+
+let cozyClient = undefined
```

Comment:
Nit: `let cozyClient` alone initializes the variable to `undefined`

Reply:
✅ I am requesting more comments. I used to be explicit but if the cozy crowd prefer being implicit, that would be fine with me